lib/zendesk_apps_tools/server.rb

```diff
@@ -44,6 +43,18 @@ class Server < Sinatra::Base
       ZendeskAppsSupport::Installed.new([app_js], [installation]).compile_js
     end
 
+    get "/:file" do |file|
+      access_control_allow_origin
+      send_file File.join(settings.root, 'assets', file)
```

Comment:
it seems like it doesn't do allow you to traverse back up the path. I don't think the filesystem would allow you to create a file called `..` though, @fbvilela might be able to confirm